packages/gravity-ui-web/src/sass/05-components/03-organisms/00-page-structure/_00-page-header.scss

```diff
@@ -7,7 +7,10 @@ $grav-page-heading-trigger-breakpoint: grav-breakpoint(medium);
 // width ($grav-page-content-max-width) themselves, where necessary.
 body > header,
 .grav-c-page-header {
+  position: sticky;
```

Comment:
Hmm... I'm not sure about doing this here. That would mean any project using Gravity will automatically get a sticky header (and it will be sticky across all screensizes too). Is that really what we want?

If this is intended for the Buildit website, then I'd suggest putting those rules in the website's own CSS. It's fine for downstream projects like the website to tweaks some of Gravity's defaults to suit their specific needs, but I'd be weary of doing something like this in Gravity itself.

That being said, this is not a hard "no". If you feel strongly this is how all Gravity page header should be, then this is the right place to do it.

Reply:
Also, whether its here or in the website's code, don't forget to give the header a solid background colour. Right now, the page content shines through and it becomes unreadable...
<img width="926" alt="image" src="https://user-images.githubusercontent.com/16718916/89655824-90a1cf80-d8c2-11ea-86ae-942a612e9d41.png">

Reply:
Correct!